index.html

```diff
@@ -204,9 +203,70 @@ <h2 class="headings">Tonics</h2>
   <a href="#">See projects</a>
 </p>
 
-</div>
+</section >
 
  <div class="card-works">
+  
+    <p class="abt-self"> About myself</p>
+
+   <p class="abt-p">
+    Hello I’m a software developer! I can help you build a product , feature or website. 
+    Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+   </p>
+
+   <p class="lets-connect">let's connect</p>
+   <div class="link">
+    <ul class="media-icons">
+      <li>
+        <a href="#">
+          <img src="./images/Github.png" alt="twitter">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Linkedin icon.png" alt="linkedin">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Mail.png" alt="mail">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Twitter.png" alt="github">
+        </a>
+      </li>
+      <li>
+       
+          <img src="./images/Vector.png" alt="vector">
+
+  </div>
+   
+    <button class="resume">
+      Get my Resume
+    </button>
+ 
+    <h3 class="lang">languages <a href="#"> <img src="./images/down-arrow.svg" class="icon1" alt=""></a> </h3>  
+     <div class="langs">
+       
+      <ul class="languages">
+        <li> <a href="#"> <img src="./images/Frame 9.png" alt=""></a> </li>
+        <li> <a href="#"><img src="./images/Frame 10.png" alt=""></a> </li>
+        <li> <a href="#"><img src="./images/Frame 11.png" alt=""></a></li>
+      </ul>
+
+    </div>
+
+      <div class="framework">
+      <p class="frameworks"> Frameworks <img src="./images/right-arrow.svg"alt="" class="icon2"> </p>
+      <p class="skills">skills <img src="./images/right-arrow.svg" alt="" class="icon2"> </p>
+  
+      </div>
```

Comment:
- [ ] [OPTIONAL] ☠️ (they are part of the requirement )kindly consider adding:

-  the   technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, each of them in a `li`
- The technology in each category(e.g., "JavasScript", "HTML" and "CSS") in a list `ul` and each of them a `li`
 as they are part of the requirement tho the previous reviewer missed it out  consider updating them to before requesting for another review to make your code professional